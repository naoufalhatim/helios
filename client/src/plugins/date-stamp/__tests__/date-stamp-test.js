import React from 'react';
import renderer from 'react-test-renderer';
import DateStamp from '../date-stamp';

const dateStampData = {
  day: 'Friday',
  date: 'August 12, 2016'
};

describe('DateStampComponent', () => {
  it('renders', () => {
    const dateStampOutput = renderer.create(
        <DateStamp
          date={dateStampData.date}
          day={dateStampData.day}/>
    );

    expect(dateStampOutput.toJSON()).toMatchSnapshot();
  });
});
