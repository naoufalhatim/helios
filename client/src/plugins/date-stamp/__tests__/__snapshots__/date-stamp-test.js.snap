exports[`DateStampComponent renders 1`] = `
<div
  className="date-stamp">
  <div
    className="label-primary">
    <span
      className="day">
      Friday
    </span>
    <span
      className="date-details">
      August 12, 2016
    </span>
  </div>
</div>
`;
